packages/thematic-lineup/package.json

```diff
@@ -14,24 +14,32 @@
   "license": "MIT",
   "scripts": {
     "build": "essex build --skipExportCheck --docs",
-    "clean": "essex clean",
-    "lint": "essex lint --fix --noFormat",
+    "clean": "essex clean docs",
+    "fix": "essex fix --formatter none",
+    "check": "essex check --formatter none",
     "release": "yarn npm publish --tolerate-republish --access public"
   },
+  "dependencies": {
+    "tslib": "^2.4.1"
+  },
   "devDependencies": {
-    "@essex/scripts": "^22.2.1",
-    "@mdx-js/react": "^1.6.22",
+    "@essex/scripts": "^23.0.4",
+    "@fluentui/font-icons-mdl2": "^8.5.6",
```

Comment:
I don't think we are using fluent in the thematic-lineup package since it is a self-contained set of ui controls